query08.sql

```diff
@@ -3,4 +3,9 @@
 */
 
 -- Enter your SQL query here
-select ...
+select start_station as most_popular_station
+from indego_trips_2019_q2
+where extract(hour from start_time) in (7,10)
```

Comment:
This will actually only match trips that have a start hour of 7 or 10, not 7 though 10. For that you could do something like:

```sql
where extract(hour from start_time) between 7 and 10
```